.github/workflows/pr-lint.yml

```diff
@@ -15,6 +15,6 @@ jobs:
     name: Validate PR title
     runs-on: self-hosted
     steps:
-      - uses: amannn/action-semantic-pull-request@v5.2.0
+      - uses: amannn/action-semantic-pull-request@v5
```

Comment:
@patlo-iog wouldn't it be better to fix version to `5.4.0` to avoid same situation in the future?

Reply:
Thanks, it's fixed. Strange how *semantic* plugin breaking on minor version upgrade :smiley: